Added 12 new Q&A pairs to the Support knowledgebase section (#1081)
wandb/docs: content/support/why_does_exporting_a_metric_as_csv_not_download_all_the_steps.md

---

> title: Why does exporting a metric as CSV not download all the steps?

Reviewer: ```suggestion
title: Why are steps missing from a CSV metric export?
```

toc_hide: true
type: docs
tags:
  - experiments
  - runs
---

> There are some export limits, so you may not be able to export the entire run history via CSV or the run.history API. To access the complete run history, download the run history artifact parquet file:

Reviewer: ```suggestion
Export limits can prevent the entire run history from being exported as a CSV or using the `run.history` API. To access the complete run history, download the run history artifact using Parquet format:
```
